### PR TITLE
Ws fix bus fault

### PR DESCRIPTION
**Pull request by grgustaf**
Still not working but I'm looking for some early review. This is working toward fixing #1458.

### CODE REVIEW

File: src/zjs_web_sockets.c [hunk 1]
```diff
@@ -173,6 +170,21 @@ static void handle_error_arg(void *unused, jerry_value_t argv[], u32_t *argc,
     *argc = 1;
 }
 
+static void emit_error(jerry_value_t obj, const char *format, ...)
```
Comment (brianjjones) [inline src/zjs_web_sockets.c]: Doesn't need to be done now, but would this be something we should put in the utils file?  Would it be useful for other modules?  Just a though.

**Comment by grgustaf** [comment]
This seems to really fix #1458 for all three test scripts now. If you increase max_payload in the scripts the client connection succeeds. However, on test-ws4-server-dhcp.js just now if I connected a few times I started seeing `(ERROR) cannot acquire send_pkt` and then an MPU fault, so you might want to see if you can reproduce that (new bug).

**Comment by brianjjones** [comment]
LGTM +1

**Comment by cuiyanx** [comment]
Verified with d04ae005e + #1513, #1458 fixed. But the test-ws4-server-dhcp.js case will throw out error after connected sometimes.
Log:
```
***** BUS FAULT *****
  Executing thread ID (thread): 0x20003ec8
  Faulting instruction address:  0x12ba6
  Precise data bus error
  Address: 0x676e6960
Fatal fault in essential thread! Spinning...

```

**Pull request merged**